excelize_test.go

```diff
@@ -1242,6 +1242,15 @@ func TestAttrValToInt(t *testing.T) {
 	assert.EqualError(t, err, `strconv.Atoi: parsing "s": invalid syntax`)
 }
 
+func TestUpdateLinkedValue(t *testing.T) {
+	// Test update the spreadsheet file.
```

Comment:
I suggest that [prepare chart sheet](https://github.com/360EntSecGroup-Skylar/excelize/blob/7ef1892f320879670c4e9a8536a70281d782625e/chart_test.go#L212-L221) in new spreadsheet file instead of add new workbook.